src/cmd/services/m3dbnode/config/fs.go

```diff
@@ -45,25 +55,25 @@ func DefaultMmapConfiguration() MmapConfiguration {
 // FilesystemConfiguration is the filesystem configuration.
 type FilesystemConfiguration struct {
 	// File path prefix for reading/writing TSDB files
-	FilePathPrefix string `yaml:"filePathPrefix" validate:"nonzero"`
+	FilePathPrefix *string `yaml:"filePathPrefix"`
```

Comment:
Should these be made private now? Or does that break the yaml parsing

Reply:
breaks YAML parsing :/